packages/node_modules/@ciscospark/widget-space/src/index.html

```diff
@@ -34,10 +34,10 @@
     <main
       class="ciscospark-space"
       id="ciscospark-widget-direct-data-api"
-      data-initial-activity="meet"
+      data-initial-activity="message"
       data-toggle="ciscospark-space"
       data-access-token="<%= process.env.CISCOSPARK_ACCESS_TOKEN %>"
-      data-to-person-email="<%= process.env.TO_PERSON_EMAIL %>"
+      data-to-person-email="nzang@cisco.com"
```

Comment:
Probably want to revert these changes :)

Reply:
Haha... yea. Gonna force push this one away